site/content/in-dev/unreleased/_index.md

```diff
@@ -56,15 +56,15 @@ or namespaces have been created yet for Catalog2 or Catalog3.
 In Polaris, you can create one or more catalog resources to organize Iceberg tables.
 
 Configure your catalog by setting values in the storage configuration for S3, Azure, or Google Cloud Storage. An Iceberg catalog enables a
-query engine to manage and organize tables. The catalog forms the first architectural layer in the [Apache Iceberg&trade; table specification](https://iceberg.apache.org/spec/#overview) and must support the following tasks:
+query engine to manage and organize tables. The catalog forms the first architectural layer in the [Apache Iceberg&trade; table specification](https://iceberg.apache.org/spec/#iceberg-table-spec) and must support the following tasks:
```

Comment:
Nit: is the anchor necessary?

Reply:
Not necessary and actually the overview anchor is better. Let me revert here.